python_modules/libraries/dagster-dbt/dagster_dbt_tests/core/test_asset_decorator.py

```diff
@@ -244,7 +244,10 @@ def my_dbt_assets(): ...
     expected_io_manager_key = DEFAULT_IO_MANAGER_KEY if io_manager_key is None else io_manager_key
 
     for output_def in my_dbt_assets.node_def.output_defs:
-        assert output_def.io_manager_key == expected_io_manager_key
+        # asset checks don't use io managers
+        if output_def.name not in my_dbt_assets.check_specs_by_output_name:
+            assert output_def.name in my_dbt_assets.keys_by_output_name
+            assert output_def.io_manager_key == expected_io_manager_key
```

Comment:
let's invert the logic to make it explicit that we're ignoring asset checks here
```suggestion
        # asset checks don't use io managers
        if output_def.name in my_dbt_assets.check_specs_by_output_name:
            continue

        assert output_def.io_manager_key == expected_io_manager_key
```